frontend/src/bundles/common/components/tabs/tab.tsx

```diff
@@ -0,0 +1,55 @@
+import clsx from 'clsx';
+import React, { type KeyboardEvent, type ReactNode,useCallback, useState } from 'react';
+
+import styles from './style.module.scss';
+
+type TabProperties = {
+    children: ReactNode;
+};
+
+const Tab: React.FC<TabProperties> = ({ children }) => {
+    const [activeTab, setActiveTab] = useState<number>(0);
+
+    const handleTabClick = useCallback((index: number) => {
+        setActiveTab(index);
+    }, [setActiveTab]);
+
+    const handleKeyDown = (event: KeyboardEvent<HTMLDivElement>, index: number): void => {
+        if (event.key === 'Enter' || event.key === 'Space') {
+            handleTabClick(index);
+        }
+    };
+
+    const onTabClick = (index: number) => () => {
+        handleTabClick(index);
+    };
+
+    const onKeyDown = (index: number) => (event: React.KeyboardEvent<HTMLDivElement>) => {
+        handleKeyDown(event, index);
+    };
+
+    return (
+        <div>
+            <div className={styles.tab__header}>
+                {React.Children.map(children, (child, index) => (
+                    <div
```

Comment:
It should be button. It would be nice if you used BaseButton component

Reply:
As I understood this components functionality has more common with [navbar](https://www.w3schools.com/howto/howto_js_topnav.asp) than [tabs](https://www.w3schools.com/howto/howto_js_tabs.asp) ? =)